packages/feedback-service/src/resolver.ts

```diff
@@ -0,0 +1,94 @@
+import { Feedback } from './schema';
+import { pubsub } from './helpers';
+import { getTransporterInstance, getEnvironmentPaths  } from "./helpers";
+  
+  
+
+const transporter = getTransporterInstance();
+const Feedback_UPDATE = "Feedback_UPDATE";
+
+export const FeedbackResolver = {
+  Subscription: {
+    // subscriptions
+    updatedFeedbackData: {
+      subscribe: () => pubsub.asyncIterator(Feedback_UPDATE)
+    },
+  },
+  Query: {
+    // queries
+    listFeedback(root: any, args: any, ctx: any) {
+      return Feedback.find()
+        .populate("module")
+        .sort({ "timestamp": -1 })
+        .exec()
+        .then(response => response)
+        .catch(err => err);
+    },
+    getFeedback(root: any, args: any, ctx: any) {
+      return Feedback.findById(args._id)
+        .populate("module")
+        .then(response => response).catch(err => err);
+    }
+  },
+  Mutation: {
+    // mutations
+    addFeedback(root: any, args: any, ctx: any) {
+      const data = new Feedback(args.input);
+      return data.save().then(response => {
+        Feedback.findById(response._id)
+          .populate("module")
+          .then(fb => {
+            if (fb) {
+              const feedbackfor = fb.portalFeedback ? "One Platfrom" : Object(fb.module).name + " module";
+              const feedbackType = fb.feedbackType === "Bug" ? "submitting a bug report" : "your feedback";
+              let body, imsg, summary;
```

Comment:
can you explain what imsg means? or probably rewrite variable names to something meaningful and that are easy to understand